python/main.py

```diff
@@ -1,139 +1,176 @@
 import json
+import settings
+import ynab_client
 
-from flask import Flask, render_template, request, jsonify, redirect
+from flask import Flask, request, jsonify, redirect
 from datetime import datetime
 from dateutil.parser import parse
 from decimal import Decimal
 
 from pynYNAB.schema.budget import Transaction
 
-import settings
-import ynab_client
-
 app = Flask('Monzo to YNAB')
 app.config['DEBUG'] = settings.flask_debug
 
-if settings.sentry_dsn:
```

Comment:
Oh, maybe it was accidently removed when we cleaned up requirements.txt. Feel free to just leave it out for now.

On a side note: i hate pythons dependancy management :D